py-glaredb/src/logical_plan.rs

```diff
@@ -60,6 +60,9 @@ impl PyLogicalPlan {
     fn to_pandas(&self, py: Python) -> PyResult<PyObject> {
         self.collect(py)?.to_pandas(py)
     }
+    fn show(&self, py: Python, show_metadata: bool, n_cols: u64) -> PyResult<()> {
+        self.collect(py)?.show(py, show_metadata, n_cols)
+    }
```

Comment:
shouldn't this be `n_rows` not `n_cols`?

Reply:
the to_string API from pyarrow supports only number of columns and not number of rows

https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.to_string

Reply:
On second thought... I think this shouldn't have any args. I think having similar functionality to duckdb's [show](https://duckdb.org/docs/api/python/relational_api) would be a good starting point.

if you want to limit the n_rows, you can just add limit x to your sql query.